pkg/reconciler/broker/controller.go

```diff
@@ -103,8 +102,8 @@ func newController(ctx context.Context, cmw configmap.Watcher, brds *brokerdeliv
 
 	brokerInformer.Informer().AddEventHandlerWithResyncPeriod(
 		cache.FilteringResourceEventHandler{
-			// Only reconcile brokers with the proper class annotation
-			FilterFunc: pkgreconciler.AnnotationFilterFunc(eventingv1beta1.BrokerClassAnnotationKey, brokerv1beta1.BrokerClass, false /*allowUnset*/),
+			// Only reconcile brokers with proper brokerclass
+			FilterFunc: reconcilerutils.BrokerClassFilter,
 			Handler:    controller.HandleAll(impl.Enqueue),
 		},
 		reconciler.DefaultResyncPeriod,
```

Comment:
Should we add the brokerclass filter in the brokercell handler to only filter the broker with googleclass at line 125?

Reply:
I think so. I remember that's part of our discussion in the chat too.

Reply:
Although I found that https://github.com/google/knative-gcp/blob/121999757c34d7fc08b03b624f09ca293e863937/pkg/client/injection/reconciler/broker/v1beta1/broker/reconciler.go#L223 here, seems like it filter out all non googleclass broker from reconcilation, but I guess we should keep this filter and the brokerInformer filter consistent?

Reply:
I think the original problem was solved by the change in brokercell. But it is always good to check if the problem persists.

Reply:
I think the plan is to eventually use `r.brokerLister.List(labels.SelectorFromSet(map[string]string{"brokercell":bc.Name, "brokercellns":bc.Namespace}` to include only brokers related to a single brokercell as commented in https://github.com/google/knative-gcp/blob/ebc58cdbe0cb1c803b059a849ee031eebd5640eb/pkg/reconciler/broker/controller.go#L118. Before this happens, another class filter in reconciler would filter the class anyways.